cmd/fyne_demo/FyneApp.toml

```diff
@@ -3,10 +3,16 @@
   Name = "Fyne Demo"
   ID = "io.fyne.demo"
   Version = "2.3.0"
-  Build = 10
+  Build = 11
 
 [Development]
   HelperText = "This binary was build with debug symbol"
 
 [Release]
   HelperText = "This binary was build without debug symbol"
+
+[LinuxAndBSD]
```

Comment:
If we make it "Linux,BSD" or "Linux&BSD" it will be machine parsable if we decide to match go's approach to this logic :).

Reply:
I tried that but it does not seem to be supported. I tried `+`, `,`, and `&`  before opening this PR but neither seem to be supported.